src/xdocs/checks/coding/illegaltokentext.xml

```diff
@@ -137,9 +141,13 @@ public void myTest() {
         </source>
         <p id="Example3-code">Example:</p>
         <source>
-public void myTest() {
+package com.puppycrawl.tools.checkstyle.checks.coding.illegaltokentext;
+
+public class Example3 {
```

Comment:
Please put comment `// xdoc section -- start` in example avoid package being in user visible snippet

Reply:
@Coding-Aliens , please fix this in separate PR.
Comment should be right above a class, to make sure package is not showing in xdoc file.

Reply:
Received